Studio/src/Analysis/AnalysisTool.cpp

```diff
@@ -144,6 +154,14 @@ std::vector<Shape::Point> AnalysisTool::get_group_difference_vectors() {
   }
   return vecs;
 }
+//---------------------------------------------------------------------------
+void AnalysisTool::mca_vanilla_pca_radio_toggled() { Q_EMIT mca_update(); }
```

Comment:
One nice thing with Qt is that signals can be directly connected to signals.  So we needn't write slots that just emit another signal.